ui/src/components/project/ReferenceAttributeForm.vue

```diff
@@ -3,44 +3,45 @@
     <header class="modal-card-head">
       <h3 class="modal-card-title">
         {{ title }}
-        <TableTag :table="table" class="is-large" />
       </h3>
     </header>
-    <section class="modal-card-body">
-      <b-field label="Table">
-        <b-select v-model="attribute.referencedTableId">
-          <option v-for="table in tables" :key="table.id" :value="table.id">{{ table.name }}</option>
+    <section class="modal-card-body reference-form">
+      <p>
+        On table <TableTag :table="table" class="is-medium" />,
+      </p>
+      <p>
+        create a link to the <label for="referenced-table"><strong>category table</strong></label>
+      </p>
+      <b-field>
+        <b-select v-model="attribute.referencedTableId" id="referenced-table">
+          <option v-for="table in linkableTables" :key="table.id" :value="table.id">{{ table.name }}</option>
         </b-select>
       </b-field>
-
-      <b-field label="Property">
-        <b-input type="text" v-model="attribute.predicateId" required />
+      <p>
+        using the <label for="property"><strong>property</strong></label>
+      </p>
+      <b-field>
+        <b-input type="text" id="property" v-model="attribute.predicateId" required />
       </b-field>
 
-      <b-field label="Identifier columns mapping" v-if="referencedTable" :addons="false">
-        <p class="help">
-          The identifier for <TableTag :table="referencedTable" /> is defined as <code>{{ referencedTable.identifierTemplate }}</code><br>
-          Which columns should be used to reference it?
+      <div v-if="referencedTable">
+        <p>
+          The identifier <code>{{ referencedTable.identifierTemplate }}</code> shall use
```

Comment:
How about "map columns" and not "use columns"?

Reply:
What do you think @ktk @l00mi ?

Reply:
Both are fine for me. But I would not waste too much time on that right now, this is something for next phase.